### ISSUE TITLE
Kopia is not listing all the snapshotted directories

### ISSUE THREAD

**Issue by sbstratos79** [kopia/kopia#3135]
A few months ago, I used kopia cli to add Google Drive as a repository and added 2 directories for snapshots. I have since used Kopia many times to restore these directories from Google Drive.
In May, I formatted my pc and didn't bother installing Kopia until now. I had backed up my kopia-ui and kopia ~/.config folders and used them to restore my Kopia installation. 
It connected with my Google Drive repo just fine. But I can only see one of the directories in the Snapshots tab.
In the Google drive folder, I can see a large amount of files which couldn't possibly all belong to the directory that is still listed in the ui. So I am assuming the backed up files of the missing directory are still present on Google drive but just not being detected.
Also, all my custom policies have disappeared.
How can I restore my directories and policies?

**Comment by sbstratos79** [comment]
Hi. I just tried it and still got the same result.

**Comment by sbstratos79** [comment]
Hi. Just wanted to know if there's anything I can do to help resolve this issue. Should I provide some logs or system information etc.?

**Comment by NickIAm** [comment]
What happens when you do `kopia snapshot list --all` from the cli? It could be that your snapshots are hidden because it only shows the snapshot directories created on the current host. Your hostname and or username probably changed after reinstalling the OS and the old snapshots are probably under a different hostname/user and hidden.

**Comment by sbstratos79** [comment]
Only the Trilium directory is listed. :(
![image](https://github.com/kopia/kopia/assets/4395586/be1bbbed-2b4b-4c7d-9480-74f6804fce4a)

I never change my username. As for the hostname, I don't think that's changed either, except for maybe  the capitalization.

**Comment by lupusA** [comment]
Hi @sbstratos79, 

that is propably the issue. If the hostname changed, then you won't see your snapshots. Even a different capitalization can cause this issue. You need to enter the same username / hostname combination at the time you created the repository and snapshotted that directory. 

The trilium directory was added afterwards?

Cheers,

**Comment by sbstratos79** [comment]
> The trilium directory was added afterwards?

Yes. But again, I'm not even sure if the capitalization has changed.

> You need to enter the same username / hostname combination at the time you created the repository and snapshotted that directory.

Should I remove the repository, change my hostname, and then add the repository again?
Is there any other way to view retrieve the snapshot?

**Comment by lupusA** [comment]
Hi @sbstratos79, 

can you run [kopia manifest list](https://kopia.io/docs/reference/command-line/advanced/manifest-list/)? It may print all the information you need. It should list all the snapshots with their username and hostname.

Cheers,

**Comment by sbstratos79** [comment]
![image](https://github.com/kopia/kopia/assets/4395586/3d23797d-33d9-45fa-853d-971a312aecbf)
It doesn't list the other directory..

**Comment by lupusA** [comment]
Hi @sbstratos79, 

in Kopia UI, selecting "All Snapshots" under the snapshot tab also does not show your other snapshot?

**Comment by sbstratos79** [comment]
No
